### ISSUE TITLE
What if L>512?

### ISSUE THREAD

**Issue by qq12cvhj** [tuan3w/spleeter-pytorch#1]
Here in the example file the L<512,so it is right.but what if the L > 512?There will be exceptions be thrown.
[mask = mask.permute([0, 3, 2, 1])[:, :, :L, 0].unsqueeze(-1)](https://github.com/tuan3w/spleeter-pytorch/blob/master/spleeter/estimator.py#L129)

**Comment by tuan3w** [comment]
Hi @qq12cvhj ,
It should be fixed in latest commit.
However, you should use this repo for learning purpose only, since it has bugs as noted in README and it only supported 2-stems model.

You should use [original repo] (https://github.com/deezer/spleeter) for production usage.
Otherwise, you might try similar projects like  [demux](https://github.com/facebookresearch/demucs) or [open-unmix](https://github.com/sigsep/open-unmix-pytorch).

**Comment by qq12cvhj** [comment]
Good job.I saved the model converted by pytorch,just to run it on mobile platforms,however tensorflow do not work on these platforms.So I choose your edition.Ok,I just use it in research work,not a productive env.I'll try it.
